integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/NextMoveServiceBus.java

```diff
@@ -42,13 +42,15 @@
 import static no.difi.meldingsutveksling.ServiceIdentifier.DPE_INNSYN;
 import static no.difi.meldingsutveksling.nextmove.ServiceBusQueueMode.DATA;
 import static no.difi.meldingsutveksling.nextmove.ServiceBusQueueMode.INNSYN;
+import static no.difi.meldingsutveksling.nextmove.ServiceBusQueueMode.MEETING;
 
 @Component
 public class NextMoveServiceBus {
 
     private static final Logger log = LoggerFactory.getLogger(NextMoveServiceBus.class);
 
     private static final String NEXTMOVE_QUEUE_PREFIX = "nextbestqueue";
+    private static final String MEETING_QUEUE_SUFFIX = "meeting";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "MEETING_QUEUE_SUFFIX" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)